src/bsp/include/bsp/IHostUart.h

```diff
@@ -1,12 +1,14 @@
 #ifndef __IHOSTUART_H__
 #define __IHOSTUART_H__
 
+#include <common/IProtobufStream.h>
 #include <cstdint>
 
-#define HOST_UART_MAX_MESSAGE_LENGTH UINT16_MAX
+#define HOST_UART_MAX_MESSAGE_LENGTH 1024
+#define HOST_UART_STREAM_SIZE 4096
```

Comment:
Should we prefer constexpr over defines? More of a general comment on our coding standards

Reply:
I don't really have an opinion on that